woocommerce/wc-admin
# Add 2col Experiment

Fixes #7635

Implements the `woocommerce_tasklist_progression_headercard_2col` experiment to decide whether users see the single column (default) or 2 columns (treatment) homepage.

### Detailed test instructions:

- Remove the `woocommerce_admin_homepage_layout` meta option if it is set.
- Apply the `woocommerce_tasklist_progression_headercard` treatment bookmarklet or update the localStorage item value.
- Confirm that homepage loads single column by default.
- Apply the `woocommerce_tasklist_progression_headercard_2col` treatment bookmarklet or update the localStorage item value.
- Confirm that homepage loads 2 columns.
